pkg/antctl/raw/multicluster/clusterset.go

```diff
@@ -0,0 +1,201 @@
+
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return err
+	}
+
+	argsNum := len(args)
+	isSingleType := false
+	if argsNum > 0 {
+		isSingleType = true
+	}
+
+	k8sClient, err := raw.AddSchemeToClientCfg(kubeconfig)
+	if err != nil {
+		return err
+	}
+
+	var clusterSets []multiclusterv1alpha1.ClusterSet
+	if isSingleType {
+		clusterSetName := args[0]
+		clusterSet := multiclusterv1alpha1.ClusterSet{}
+		err = k8sClient.Get(context.TODO(), types.NamespacedName{
+			Namespace: optionsClusterSet.namespace,
+			Name:      clusterSetName,
+		}, &clusterSet)
+		if err != nil {
+			return err
+		}
+		gvks, unversioned, err := k8sClient.Scheme().ObjectKinds(&clusterSet)
+		if err != nil {
+			return err
+		}
+		if !unversioned && len(gvks) == 1 {
+			clusterSet.SetGroupVersionKind(gvks[0])
+		}
+		clusterSets = append(clusterSets, clusterSet)
+	} else {
+		clusterSetList := &multiclusterv1alpha1.ClusterSetList{}
+		err = k8sClient.List(context.TODO(), clusterSetList, &client.ListOptions{Namespace: optionsClusterSet.namespace})
+		if err != nil {
+			return err
+		}
+		clusterSets = clusterSetList.Items
+	}
+
+	if len(clusterSets) == 0 {
+		if optionsClusterSet.namespace != "" {
+			fmt.Printf("No resource found in Namespace %s\n", optionsClusterSet.namespace)
+		} else {
+			fmt.Println("No resource found in all Namespaces")
+		}
+		return nil
+	}
+
+	switch optionsClusterSet.outputFormat {
+	case "json":
+		prettyJson, err := jsonOutput(clusterSets)
+		if err != nil {
+			return err
+		}
+		fmt.Println(prettyJson.String())
+	case "yaml":
+		yamlByte, err := yamlOutput(clusterSets)
+		if err != nil {
+			return err
+		}
+		fmt.Println(string(yamlByte))
+	default:
+		if len(args) != 0 {
+			fmt.Println(outputClusterSet(clusterSets))
+		} else {
+			fmt.Println(outputAllClusterSet(clusterSets))
+		}
+	}
+
+	return nil
+}
+
+func outputClusterSet(clusterSets []multiclusterv1alpha1.ClusterSet) string {
+	var output strings.Builder
+
+	formatter := "%-25s%-25s%-25s%-28s%-20s%-10s\n"
+	output.Write([]byte(fmt.Sprintf(formatter, "CLUSTERID", "NAMESPACE", "CLUSTERSETID", "TYPE", "STATUS", "REASON")))
+	sort.SliceStable(clusterSets, func(i, j int) bool {
+		return strings.Compare(clusterSets[i].Namespace, clusterSets[j].Namespace) == 1
+	})
+	for _, clusterSet := range clusterSets {
+		sort.SliceStable(clusterSet.Status.ClusterStatuses, func(i, j int) bool {
+			return strings.Compare(clusterSet.Status.ClusterStatuses[i].ClusterID, clusterSet.Status.ClusterStatuses[j].ClusterID) == 1
+		})
+		for _, r := range clusterSet.Status.ClusterStatuses {
+			for _, condition := range r.Conditions {
+				fmt.Fprintf(&output, formatter, r.ClusterID, clusterSet.Namespace, clusterSet.Name, condition.Type, condition.Status, condition.Reason)
+			}
+		}
+	}
+
+	return output.String()
+}
+
+func outputAllClusterSet(clusterSets []multiclusterv1alpha1.ClusterSet) string {
+	var output strings.Builder
+	formatter := "%-25s%-20s%-20s%-20s%-15s%-10s\n"
```

Comment:
you need to make the length longer for `OVERALL_CONDITION` column, I got below output:
```
CLUSTERSETID             NAMESPACE           TOTAL_CLUSTERS      READY_CLUSTERS      OVERALL_CONDITIONREASON
```